content/v1.13-docs/configuration/acme/dns01/route53.md

```diff
@@ -184,6 +184,8 @@ spec:
 
 Note that, as mentioned above, the pod is using `arn:aws:iam::XXXXXXXXXXX:role/cert-manager` as a credentials source in Account X, but the `ClusterIssuer` ultimately assumes the `arn:aws:iam::YYYYYYYYYYYY:role/dns-manager` role to actually make changes in Route53 zones located in Account Y.
 
+**Note:** If you are using IRSA, omit `route53.role`. See [#4053](https://github.com/cert-manager/cert-manager/issues/4053).
```

Comment:
Move this note to the `EKS IAM Role for Service Accounts (IRSA)` (below).
 * https://cert-manager.io/docs/configuration/acme/dns01/route53/#eks-iam-role-for-service-accounts-irsa

And update the example ClusterIssuer config in that IRSA section:
 * https://cert-manager.io/docs/configuration/acme/dns01/route53/#issuerclusterissuer-config

I don't know much about AWS and IRSA....can you explain (in the documentation note) **why** the role does not need to be supplied in this case.

The documentation for the Go Web SDK suggests that RoleARN **is** required
   * https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/credentials/stscreds#AssumeRoleOptions
   * https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/credentials/stscreds#WebIdentityRoleOptions

I'm interested to understand this because I'm currently working on the following update to the Route53 solver code in cert-manager: 
 * https://github.com/cert-manager/cert-manager/pull/7236

Reply:
@wallrj thank you for reviewing this! I must admit that - just like you - I don't know much about AWS and IRSA. The reason I opened this PR was [the positive feedback on the comment demonstrating "the solution"](https://github.com/cert-manager/cert-manager/issues/4053#issuecomment-892720234).

For now I reckon the best course of action is to close this PR, because I am afraid I am unable to back it up beyond said comment.